config/content.d/staging.horse.json

```diff
@@ -5,7 +5,8 @@
       "/docs/private-cloud/release-notes/": "https://github.com/rcbops/rpc-openstack/master/",
       "/docs/api-doc-template/": "https://github.com/rackerlabs/docs-common/",
       "/docs/billing-api/": "https://github.com/rackerlabs/docs-billing-apiguide/",
-      "/docs/sdp/": "https://github.com/rackerlabs/docs-SDP/"
+      "/docs/sdp/": "https://github.com/rackerlabs/docs-SDP/",
+      "/docs/managed-vmware-services/server-virt-customer-handbook/":"https://github.com/rackerlabs/rpc-vmware/server-virt-handbook/master/"
```

Comment:
@asettle, remove `master/` from the end of the value in this key:value pair. This value should match the `contentIDBase` value from [the `_deconst.json` file](https://github.com/rackerlabs/rpc-vmware/pull/152/files#diff-8164509f5fbf24b88593c25f3fe77e77).